conf/messages

```diff
@@ -120,8 +117,7 @@ hts.create.account.terms.ul1-li3-unknown=they will tell us each time they leave
 hts.create.account.terms.ul1-li4=they are creating a Help to Save account for the first time
 hts.create.account.terms.p2=Ask the customer if they understand and agree to the terms and conditions, and want to create a Help to Save account.
 
-hts.account.exists.title=Customer already has an account
-hts.account.exists.h1=Customer already has an account
+hts.account.exists.title.h1=Customer already has an account
 hts.account.exists.p1=Customer can pay in by bank transfer or standing order. They will need to set up the payment through their bank.
 
 hts.account-created.title=Help to Save account created
```

Comment:
hts.account-created.title will change with the above comment

Reply:
yep